release/test_release_gold.sh

```diff
@@ -16,6 +18,7 @@ function set_up {
 	export LIFERAY_COMMON_EXIT_CODE_SKIPPED=4
 	export LIFERAY_RELEASE_REPOSITORY_OWNER="natocesarrego"
 	export LIFERAY_RELEASE_VERSION="test-tag"
+	export _PRODUCT_VERSION="2024.q2.11"
```

Comment:
@kiwm, can you set this variable on the beginning of `test_is_update_release_applicable`?

Ideally, we should add to the `set_up` only the resources in common for all test. :slightly_smiling_face: